test_requirements/base.txt

```diff
@@ -13,7 +13,7 @@ django-appdata>=0.1.4
 django-classy-tags
 # django-cms>=3.0.12
 django-filer
-django-mptt<0.7.0
+django-mptt<0.7.4
```

Comment:
I know that 0.7.0/0.7.1 broke the CMS, but apparently 0.7.3+ are safe and it's the only way to support Django 1.8. Test suite runs ok, so I'd accept it (mptt breakages are usually very obvious and final)